spec/models/ems_refresh/refreshers/openshift_refresher_spec.rb

```diff
@@ -104,8 +104,7 @@ def assert_specific_container_route
                                 :ems_ref      => "ff8a8e45-e385-11e4-9d96-f8b156af4ae1",
                                 :name         => "route-edge",
                                 :namespace    => "default",
-                                :host_name    => "www.example.com",
-                                :service_name => "frontend"
+                                :host_name    => "www.example.com"
```

Comment:
I don't understand how you would get the name starting with the container routes.
The columns are:
kube=# select * from container_routes;
 id |               ems_ref                |    name    | creation_timestamp  | resource_version | namespace |    host_name    | path | ems_id | container_service_id